src/metaTube.cxx

```diff
@@ -1102,20 +1102,20 @@ MetaTube::M_Write()
   }
   delete[] pntVal;
 
-  int posId = M_GetPosition("id", positionUsed);
-  int posX = M_GetPosition("x", positionUsed);
-  int posY = M_GetPosition("y", positionUsed);
-  int posZ = M_GetPosition("z", positionUsed);
-  int posRed = M_GetPosition("red", positionUsed);
-  int posGreen = M_GetPosition("green", positionUsed);
-  int posBlue = M_GetPosition("blue", positionUsed);
-  int posAlpha = M_GetPosition("alpha", positionUsed);
-  int posMark = M_GetPosition("mark", positionUsed);
+  unsigned int posId = M_GetPosition("id", positionUsed);
```

Comment:
These unsigned changes are reverted in the next commit. Perhaps this commit should be squashed with the next to reduce noise?

Reply:
Merged as requested.